ui/webpack.common.ts

```diff
@@ -29,14 +30,29 @@ module.exports = {
     },
     extensions: ['.tsx', '.ts', '.js', '.wasm'],
   },
+  node: {
```

Comment:
i know this was required for the vendor build, but is it required for the dist build?

Reply:
yup